docs/ops/comparison/Equal_1.md

```diff
@@ -13,39 +22,31 @@
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
```

Comment:
@rkazants , @lazarevevgeny is equal op supporting PDPD broadcast?

Reply:
I don't think this link will work when HTML will be generated from this markdown file.

Reply:
```suggestion
    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](https://docs.openvinotoolkit.org/latest/openvino_docs_ops_broadcast_rules.html),
```

Reply:
The right approach is to use `[Broadcast Rules For Elementwise Operations](../broadcast_rules.md)`. 

We should never hardcode links to https://docs.openvinotoolkit.org/latest/openvino_docs_ops_broadcast_rules.html

Reply:
@nosovmik , @mvafin do you know if "Equal" op supports PDPD type of broadcasting?

@sdurawa , anyway, for the consistency with other elementwise ops I would recommend to keep the pdpd in the spec

Reply:
@lazarevevgeny Can you explain to me why this approach is the right one? Are we expecting users to be browsing .md files rather than the docs OpenVINO is hosting? If so, why are we using tex in the `.md` files?

Reply:
When the documentation is built the MD file is replaced with a proper html page. Take a look at other OV documentation and you will see that we use `.md` everywhere.